### ISSUE TITLE
Envy cannot edit deeply nested files:

### ISSUE THREAD

**Issue by andrewgross** [shaunvxc/envy#5]
Trying to edit a file deep in `aws-cli`:

```
envy edit awscli/awscli/customizations/datapipeline/createdefaultroles.py
```

However, when it edits the file it opens up a different one based just on the last split on `/` of the file path.

```
~/.envs/sqoop/lib/python2.7/site-packages/awscli/createdefaultroles.py
```

**Comment by shaunvxc** [comment]
Yea, I think this is our issue is on line 71 (inside `edit`): 

`file_path = args.path[0].split("/")[-1]`

**Comment by andrewgross** [comment]
Fixed in PR #6

**Comment by shaunvxc** [comment]
Thanks!
